src/components/calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+function Calculator() {
+  // eslint-disable-next-line react/button-has-type
```

Comment:
Kindly avoid disabling linter checks inline. Linters represent a coding guideline you share with your team and most of the time enforce best coding practices.